src/components/footer/www/footer.jsx

```diff
@@ -108,7 +124,7 @@ const Footer = props => (
                         </a>
                     </dd>
                     <dd>
-                        <a href="https://en.scratch-wiki.info/">
+                        <a href="{wikilink}">
```

Comment:
```suggestion
                        <a href="{wikiLink}">
```
(to match with other suggested code)